devito/passes/iet/misc.py

```diff
@@ -104,9 +113,88 @@ def relax_incr_dimensions(iet, **kwargs):
             # `i.dim.symbolic_max = x0_blk0 + x0_blk0_size - 1` then the generated
             # maximum will be `MIN(x0_blk0 + x0_blk0_size + 1, x_M + 2)`
 
+            # Step 1: Interval offset care
+            # Here we create two dictionaries to keep track of offsets incurring in
+            # 'i' and 'i.root' iteration
+            min_map = {}  # For the Iteration minimum
```

Comment:
just 

```
min_map = {
     i.dim.root.symbolic_min: root_min,
     i.dim.symbolic_min: i.symbolic_min
}
```

isn't it more readable like this?

Reply:
regardless, this definition line alone isn't really useful

Reply:
>regardless, this definition line alone isn't really useful

Which line? Do not really get it. (?)